target/linux/ramips/mt76x8/base-files/etc/board.d/02_network

```diff
@@ -69,6 +69,7 @@ ramips_setup_interfaces()
 		ucidef_add_switch "switch0" \
 			"1:lan" "0:wan" "6@eth0"
 		;;
+	asus,rt-ac1200|\
```

Comment:
Please move this block to follow the alphabetical order.

Reply:
fixed